bot/resources/tags/venv.md

```diff
@@ -0,0 +1,18 @@
+**Virtual Environments**
+
+Virtual environments are isolated Python environments, which make it easier to keep your system clean and manage dependencies. By default, when activated, only libraries and scripts installed in the virtual environment are accessible.
+
+To create a new virtual environment, you can use the `venv` module: `python -m venv .venv`
```

Comment:
```suggestion
To create a new virtual environment, you can use the standard library `venv` module: `python3 -m venv .venv` (replace `python3` with `python` or `py` on windows)
```
May need `python3` on unix, so probably worth changing to something like this. Also mention that's it's in the standard library